doc/en/conf.py

```diff
@@ -141,10 +141,18 @@
 linkcheck_ignore = [
     "https://blogs.msdn.microsoft.com/bharry/2017/06/28/testing-in-a-cloud-delivery-cadence/",
     "http://pythontesting.net/framework/pytest-introduction/",
-    r"https://github.com/pytest-dev/pytest/issues/\d+",
-    r"https://github.com/pytest-dev/pytest/pull/\d+",
+    r"https://pypi\.org/project/pytest.*",
+    r"https://github\.com/sponsors/.*",
+    r"https://github\.com/pytest-dev/pytest/issues/.*",
```

Comment:
This should probably be more accurate. Such a wildcard would also silence obviously broken URLs like https://github.com/pytest-dev/pytest/pull/zombie that should produce errors otherwise.